mozilla_schema_generator/glean_ping.py

```diff
@@ -329,7 +329,12 @@ def generate_schema(self, config, generic_schema=False) -> Dict[str, Schema]:
             else:
                 generated = super().generate_schema(new_config)
                 for schema in generated.values():
-                    schema.schema.update(defaults)
+                    # We want to override each individual key with assembled defaults,
+                    # but keep values _inside_ them if they have been set in the schemas.
+                    for key, value in defaults.items():
+                        if key not in schema.schema:
+                            schema.schema[key] = {}
+                        schema.schema[key].update(value)
```

Comment:
Why is this needed?
Should we add a test to cover this change?

Reply:
Because in https://github.com/mozilla-services/mozilla-pipeline-schemas/pull/793 we set _some_ values but want to override some of them.

Yes, I should add a test.